store/state/base.go

```diff
@@ -0,0 +1,54 @@
+package state
+
+import (
+	//	"github.com/tendermint/tendermint/crypto/merkle"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+type Base struct {
```

Comment:
can you also add all the missing godoc comments to understand better the changes from this PR?

Reply:
^^